<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealWidgetClass</class>
 <widget class="QDialog" name="DealWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DealWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color:rgb(55, 107, 113);
}</string>
  </property>
  <widget class="QLabel" name="dealLbl">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название сделки</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>32</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="statusSaleLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Этап сделки:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Новая заявка</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Переговоры</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Принятие решения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Согласование договора</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Завершить сделку</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="sumLayout">
        <item>
         <widget class="QLabel" name="sumLbl">
          <property name="text">
           <string>Сумма</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sum">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="timeDealLayout">
        <item>
         <widget class="QLabel" name="timeDealLbl">
          <property name="text">
           <string>Сроки сделки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeDeal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="sourceDealLayout">
        <item>
         <widget class="QLabel" name="sourceDealLbl">
          <property name="text">
           <string>Источник сделки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sourceDeal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="responsibleLayout">
        <item>
         <widget class="QLabel" name="responsibleLbl">
          <property name="text">
           <string>Ответственный</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="responsible">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="clientLbl">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Клиент</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="client">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="phoneCallBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Звонок</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="smsBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>смс</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="numberLbl">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Номер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="number">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="emailLbl">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="email">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
